Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackThemeConfiguration.swift

```diff
@@ -129,17 +123,22 @@ public class SentryUserFeedbackThemeConfiguration: NSObject {
     /**
      * - note: We need to keep a reference to a default instance of this for comparison purposes later. We don't use the default to give UITextFields a default style, instead, we use `UITextField.BorderStyle.roundedRect` if `SentryUserFeedbackThemeConfiguration.outlineStyle == defaultOutlineStyle`.
      */
-    let defaultOutlineStyle = OutlineStyle()
+    let defaultOutlineStyle = SentryFormElementOutlineStyle()
     
     /**
      * Options for styling the outline of input elements and buttons in the feedback form.
      */
-    public lazy var outlineStyle: OutlineStyle = defaultOutlineStyle
+    public lazy var outlineStyle: SentryFormElementOutlineStyle = defaultOutlineStyle
     
     /**
      * Background color to use for text inputs in the feedback form.
      */
     public var inputBackground: UIColor = UIColor.secondarySystemBackground
+    
+    /**
+     * Background color to use for text inputs in the feedback form.
+     */
+    public var inputForeground: UIColor = UIScreen.main.traitCollection.userInterfaceStyle == .dark ? UIColor.lightText : UIColor.darkText
```

Comment:
I'd forgotten to add this to the configuration options, realized it going through the docs pr